classes/options-general.php

```diff
@@ -821,20 +821,20 @@ class="radio-thumbnail<?php if ( 'TERRAIN' == $map_type ) {
 
 													<tr valign="top">
 														<td><label
-																for="map_stylesheet"><?php _e( 'Theme', 'simplemap' ); ?></label>
+																for="map_stylesheet"><?php _e( 'Theme', 'SimpleMap' ); ?></label>
 														</td>
 														<td>
 															<select name="map_stylesheet" id="map_stylesheet">
 																<?php
-																echo '<optgroup label="' . __( 'Default Themes', 'simplemap' ) . '">' . "\n";
+																echo '<optgroup label="' . __( 'Default Themes', 'SimpleMap' ) . '">' . "\n";
 																foreach ( $themes1 as $file => $name ) {
 																	$file_full = 'inc/styles/' . $file;
 																	echo '<option value="' . esc_attr( $file_full ) . '" ' . selected( $map_stylesheet, $file_full, false ) . '>' . esc_attr( $name ) . '</option>' . "\n";
 																}
 																echo '</optgroup>' . "\n";
 
 																if ( ! empty( $themes2 ) ) {
-																	echo '<optgroup label="' . __( 'Custom Themes', 'simplemap' ) . '">' . "\n";
+																	echo '<optgroup label="' . __( 'Custom Themes', 'SimpleMap' ) . '">' . "\n";
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 144 characters

<!-- Comment by [SideCI](https://sideci.com) -->